proximo-server/main.go

```diff
@@ -111,9 +111,21 @@ func main() {
 			Desc:   "maximum number of unacknowledged messages",
 			EnvVar: "PROXIMO_NATS_MAX_INFLIGHT",
 		})
+		pingIntervalSeconds := cmd.Int(cli.IntOpt{
+			Name:   "ping-interval",
+			Value:  3,
+			Desc:   "interval in seconds for connection pings",
+			EnvVar: "PROXIMO_PING_INTERVAL_SECONDS",
```

Comment:
It would change this to something like this to match the other environment variables.
```suggestion
			EnvVar: "PROXIMO_NATS_PING_INTERVAL_SECONDS",
```
Same for the number of timeouts...